### ISSUE TITLE
[FEATURE]  invoke tpcds from sql 

### ISSUE THREAD

**Issue by zinking** [apache/incubator-kyuubi#1915]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the feature

using sql extensions to create and run tpcds sql benchmarks

```
call  spark.system.create_tpcds(sf=10, db='tpcds_sf10', fmt='parquet', parallel=20)
call  spark.system.run_tpcds(db='tpcds_sf10', filter='q1-v2.4')

```

### Motivation

 users with no dev background can create and run tpcds datasets

### Describe the solution

```
call  spark.system.create_tpcds(sf=10, db='tpcds_sf10', fmt='parquet', parallel=20)
call  spark.system.run_tpcds(db='tpcds_sf10', filter='q1-v2.4')

```

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

**Comment by zinking** [comment]
@yaooqinn  if you are okay with this approach, I will spend sometime and contribute a patch

**Comment by yaooqinn** [comment]
I am OK, can you detail a little about the user behavior, what are we gonna change?

**Comment by zinking** [comment]
nothing really changes from the user side, it's just they can now issue these *control* command from the beeline.
implementation side we will introduce store procedure like routines, which I think is perfect for this task.  also more control commands could be supported from this on.

**Comment by pan3793** [comment]
Tips: currently the tpcds moudle can not be ship with the official release because of license, we may need to replace it by a Apache licensed one https://github.com/Teradata/tpcds

**Comment by yaooqinn** [comment]
we currently support scala operation like `spark.sql(...)/ spark.udf.`, why we need the `call` prefix

**Comment by zinking** [comment]
> we currently support scala operation like `spark.sql(...)/ spark.udf.`, why we need the `call` prefix

that's true, now kyuubi is supporting scala code, `call` is less relevant from a developer context. but at the time of writing, I was more thinking of doing this in pure sql context.

**Comment by zinking** [comment]
> Tips: currently the tpcds moudle can not be shipped with the official release because of license, we may need to replace it by an Apache licensed one https://github.com/Teradata/tpcds

@pan3793 I did remember there are some packages related to tpcds in kyuubi before, seems no longer there anymore? when is it removed?

**Comment by pan3793** [comment]
We have no plan to remove the TPC-DS related classes/modules, but it can NOT be included into the official source/binary release bundle because of the license.
